src/styles/blocks/main-content.scss

```diff
@@ -0,0 +1,24 @@
+.main {
```

Comment:
you can remove `margin-right` from `card` and add `justify-content: space-between` to `main__content` as you've already hardcoded it's width